python/tests/test_indexmanagement.py

```diff
@@ -0,0 +1,71 @@
+import unittest
```

Comment:
Can we run this as a part of azure pipieline?

Reply:
Thanks. Adding stage for running the python test in pipeline